.github/workflows/nightly.yml

```diff
@@ -90,10 +90,10 @@ jobs:
         if: ${{ matrix.os == 'windows-latest' }}
       - name: Building dev version
         run:
-          cargo build -Z build-std=std,core,alloc,panic_abort --target x86_64-unknown-hermit
+          cargo build -p rusty_demo -Z build-std=std,core,alloc,panic_abort --target x86_64-unknown-hermit
       - name: Building release version
         run:
-          cargo build -Z build-std=std,core,alloc,panic_abort --target x86_64-unknown-hermit --release
+          cargo build -p rusty_demo -Z build-std=std,core,alloc,panic_abort --target x86_64-unknown-hermit --release
```

Comment:
Why are you only building rusty_demo?

Reply:
It is currently the only test, which is used.

Reply:
That is only true in this PR. The one you just merged added a standalone pi sequential test

Reply:
But the standalone pi sequential test is also included in rusty_demo

Reply:
> But the standalone pi sequential test is also included in the package `rusty_demo`. Consequently, we build also the standalone version of the pi sequential test.

No, you don't. If I change something in the sequential pi test that does not compile, you will not notice it with this CI pipeline. Furthermore, if you change something in the demo and not adapt the sequential pi test, you may break the standalone test without realizing it. We can remove the standalone tests again, but even than I fail to see the benefit of that change. 🤷